recipes/tesseract/all/conanfile.py

```diff
@@ -92,13 +112,21 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
         # remove cmake
         tools.rmdir(os.path.join(self.package_folder, 'cmake'))
+        # required for 5.0
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
```

Comment:
there's no 5.0 🤔

Reply:
I was experimenting in packaging 5.0.x-alpha. Of course, I can drop it now and add for the final version.